docs/genai/tutorials/phi3-v.md

```diff
@@ -152,3 +150,42 @@ Support for Windows machines with GPUs other than NVIDIA is coming soon!
    The products include Chocolade, Gummibarchen, Scottish Longbreads, Sir Rodney's Scones, Tarte au sucre,
    and Chocolate Biscuits. The Grand Total column sums up the sales for each product across the two quarters.</s>
    ```
+
+## Run with DirectML
```

Comment:
Should we re-order the examples to first show DML, then CUDA, and finally CPU to match the recommended order in the [model cards](https://huggingface.co/microsoft/Phi-3-vision-128k-instruct-onnx-directml#onnx-models)?

Reply:
Yes, we could do that. I played with the order

Reply:
Actually I'll finish this PR with the original order to keep it simple then submit a new one with a simple re-order